Directory.Build.props

```diff
@@ -124,4 +124,13 @@
   <PropertyGroup>
     <CustomAfterMicrosoftCommonTargets>$(RepoRoot)build\AfterCommonTargets.targets</CustomAfterMicrosoftCommonTargets>
   </PropertyGroup>
+  
+  <!-- API Compat -->
```

Comment:
Can these properties go into `src\Directory.Build.props` instead? Grouping them with the `RunApiCompat` property would make sense IMO.

Reply:
Yes I'll do that!

---
In reply to: [280106010](https://github.com/dotnet/machinelearning/pull/3623#discussion_r280106010) [](ancestors = 280106010)